app/src/main/java/org/fossasia/openevent/general/auth/LoginFragment.kt

```diff
@@ -137,6 +137,7 @@ class LoginFragment : Fragment() {
 
         rootView.forgotPassword.setOnClickListener {
             hideSoftKeyboard(context, rootView)
+            (activity as AppCompatActivity).supportActionBar?.hide()
```

Comment:
Use kotlin smart cast instead. And full screen means bottom navigation should also hide.

Reply:
And make sure to reset the screen when tick or back button is pressed.